src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/MethodTable.Runtime.cs

```diff
@@ -98,6 +98,14 @@ internal static bool BothSimpleCasting(MethodTable* pThis, MethodTable* pOther)
             return ((pThis->_uFlags | pOther->_uFlags) & (uint)EETypeFlags.ComplexCastingMask) == 0;
         }
```

Comment:
You can delete FastEquals on EETypePtr. Array.Copy was the only method that used it.